src/Components/Blazor/Build/testassets/razorclasslibrary/ExampleJsInterop.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.JSInterop;
+using System.Threading.Tasks;
+
+namespace RazorClassLibrary
+{
+    public class ExampleJsInterop
+    {
+        public static ValueTask<string> Prompt(IJSRuntime jsRuntime, string message)
```

Comment:
Can we remove this type? Minimizes the size of these test project files.

Reply:
Removed all the files except the content